hobbymatcher/src/main/java/com/hobbymatcher/controller/hobby/HobbyController.java

```diff
@@ -0,0 +1,67 @@
+package com.hobbymatcher.controller.user;
+
+import com.hobbymatcher.entity.Hobby;
+import com.hobbymatcher.service.HobbyService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.SessionAttributes;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@SessionAttributes()
```

Comment:
Can we add the `@CrossOrigin` annotation here?

Reply:
got it, I will add it right away